CONTRIBUTORS.md

```diff
@@ -32,6 +32,48 @@ This project and everyone participating in it is governed by this [Code of Condu
 - For consensus breaking changes, branch off of the `next` branch.
 - For hotfixes, branch off of `master`.
 
+### GitHub Tags
+We use tags on PRs and issues to categorize them. You should try to select a tag from each category below if possible. 
+
+**Timeline- and consensus- based tags:**
+- Next non-breaking change => `ship now`
+- Next breaking change => `ship next`
+- Future change, in the work pipeline => `ship future`
+- Issues that are not being worked on (and there are no plans to) => `icebox`
+- PRs that are temporarily frozen => `frozen`
+- Must be in a breaking change => `consensus-critical`
+
+**Issue Type Tags:**
+- For first-time or new contributors => `good first issue`
+- Stacks lab issues (infrastructure-type changes) => `help wanted`
+- Documentation updates => `documentation`
+- Improvement to the chain (speed, reliability) => `optimization`
+- Improvements to CI/CD => `CI`
```

Comment:
"devops"?

Reply:
I think `CI` is better than `devops` here.

Reply:
CI does not include CD?
Should we have separate CD?

Reply:
I don't think it's worth it to have a separate CD tag. I'm going to leave this as CI.